rust/cubestore/cubestore/src/remotefs/minio.rs

```diff
@@ -207,25 +208,32 @@ impl RemoteFs for MINIORemoteFs {
             let path = self.s3_path(remote_path);
             info!("path {}", remote_path);
             let bucket = self.bucket.read().unwrap().clone();
-            let status_code = cube_ext::spawn_blocking(move || -> Result<u16, CubeError> {
-                let (mut temp_file, temp_path) =
-                    NamedTempFile::new_in(&downloads_dir)?.into_parts();
 
-                let res = bucket.get_object_stream_blocking(path.as_str(), &mut temp_file)?;
-                temp_file.flush()?;
+            let temp_dir = tempdir_in(downloads_dir)?;
```

Comment:
I'd avoid changing `NamedTempFile::new_in()` allocation by using `tempdir_in()` as logic isn't equivalent and will require additional regression testing.